packages/irmin/irmin.2.3.0/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+maintainer:   "thomas@gazagnaire.org"
+authors:      ["Thomas Gazagnaire" "Thomas Leonard"]
+license:      "ISC"
+homepage:     "https://github.com/mirage/irmin"
+bug-reports:  "https://github.com/mirage/irmin/issues"
+dev-repo:     "git+https://github.com/mirage/irmin.git"
+doc:          "https://mirage.github.io/irmin/"
+
+build: [
+ ["dune" "subst"] {pinned}
+ ["dune" "build" "-p" name "-j" jobs]
+ ["dune" "runtest" "-p" name] {with-test}
```

Comment:
If there is a change to make, would it be possible to change the `{pinned}` to `{dev}` and to add `-j jobs` to the tests in all of the opam files?

Reply:
Done :+1: